package.json

```diff
@@ -138,12 +137,10 @@
     "configstore": "^3.1.1",
     "cssstyle": "1.2.1",
     "details-element-polyfill": "2.2.0",
-    "esprima": "^4.0.1",
```

Comment:
still used by `collect-strings.js`, but no longer in `unminified-javascript`, so just a devDependency